frontend/src/v5/ui/routes/dashboard/projects/calibration/calibrationHeader/calibrationHeader.component.tsx

```diff
@@ -29,9 +32,20 @@ const STEPS = [
 ];
 
 export const CalibrationHeader = () => {
-	const { step, setStep, isStepValid, origin } = useContext(CalibrationContext);
+	const { teamspace, project } = useParams();
+	const { step, setStep, isStepValid, origin, vector3D, drawingId } = useContext(CalibrationContext);
 	const isLastStep = step === 2;
 
+	const getIsStepValid = () => {
+		if (step === 0) return !!(vector3D.start && vector3D.end);
```

Comment:
Maybe this is for a later issue, but perhaps we should also check that the start and end point are not equal

Reply:
Fair point. Then we should implement the same on the 2d vector as well!